notebooks/docker_and_kubernetes/solutions/1_intro_docker.ipynb

```diff
@@ -8,8 +8,8 @@
     "\n",
```

Comment:
<p>Added this authorization command where <code>docker</code> command is needed. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/428/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>